docs/firefighting/index.md

```diff
@@ -257,25 +257,27 @@ $ mount /mnt/shared_data
 # verify that it is mounted and that there are files in the subfolders
 ```
 
-### Forcing re-connection of an NFS client in a webworker
+### Forcing re-connection of an NFS client in a webworker (or other commcarehq machine)
 
 It may happen, specially if the client crashes or has kernel oops, that a connection gets in a state where it cannot be broken nor re-established.  This is how we force re-connection in a webworker.
+
 1. Verify NFS is actually stuck
-    a. “df” doesn’t work, it hangs. Same goes for lsof.
-    b. “umount” results in umount.nfs: /mnt/shared_icds: device is busy
-2. top all gunicorns and supervisor
-    a. `sudo supervirsorctl stop all`
-    b. jVerify “stuck” processes keep running: $ ps aux|grep gunicorn
-    c. $ sudo service supervisord stop
-    d. Verify “stuck” processes are gone: $ ps aux|grep gunicorn
+    1. `df` doesn’t work, it hangs. Same goes for `lsof`.
+    2. `umount` results in `umount.nfs: /mnt/shared_icds`: device is busy
+2. top all app processes (gunicorn, etc) and datadog
+    1. `sudo supervisorctl stop all`
+    5. `sudo service datadog-agent stop`
```

Comment:
Curious why this is a 5? I thought you could write ordered lists with `1.` starting each item and then markdown takes care of the numbering - this item shows up as `ii` in the preview, not `v`.

Reply:
Oops yeah that was a mistake. I changed it to `2`. But yeah you're right it replaces any number with numbers in the right order (1, 2, 3, etc. / i, ii, iii, etc.).